src/main/java/com/infobip/interviewdemo/api/CrudController.java

```diff
@@ -0,0 +1,31 @@
+package com.infobip.interviewdemo.api;
+
+import com.infobip.interviewdemo.domain.CommentEntity;
+import com.infobip.interviewdemo.dto.CommentRequest;
+import com.infobip.interviewdemo.service.CommentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("comment")
+@RequiredArgsConstructor
+public class CrudController {
+
+    private final CommentService commentService;
+
+    @PostMapping(produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseStatus(HttpStatus.CREATED)
```

Comment:
```suggestion
    @ResponseStatus(CREATED)
```
If you would stick with this kind of approach, at least use static import so it will be more cleaner and readable.